src/store/Developers.js

```diff
@@ -49,7 +50,23 @@ const actions = {
   }
 };
 
-const getters = {};
+const getters = {
+  newDevelopers(state) {
+    const newDevs = clone(state.developers).filter(dev => {
+      const lastMonthDate = new Date();
+      const ONE_MONTH = 30;
+      lastMonthDate.setDate(lastMonthDate.getDate() - ONE_MONTH);
+      return +new Date(dev.createdAt) >= +lastMonthDate;
```

Comment:
There's no need to convert to number, dates can be compared with the normal comparison operators without doing the conversion.

Reply:
Right